snap/snapcraft.yaml

```diff
@@ -966,22 +1035,16 @@ parts:
 
   # SECURITY SERVICES PARTS
   vault:
-    after: [go]
+    after: [go112]
     plugin: make
     source: https://github.com/hashicorp/vault.git
-    source-tag: v1.0.2
+    source-tag: v1.3.1
     source-depth: 1
     override-build: |
-      # When the snap is built by Jenkins w/in a docker instance, the existence of
-      # the top level go.mod file causes the consul build to fail when using go 1.13.
-      # This workaround can probably be removed if/when consul is updated to a more
-      # recent version.
-      if [ -f /build/go.mod ]; then
-        mv /build/go.mod /build/go.mod.bk
-      fi
+      patch -p1 < "$SNAPCRAFT_PROJECT_DIR/snap/local/patches/0001-Makefile-rm-golangci-lint-tool.patch"
```

Comment:
I would still like to see a comment about why we need this patch and a link to the upstream issue here